dev/NumberBox/NumberBox.xaml

```diff
@@ -205,7 +213,6 @@
             <Setter Property="AutomationProperties.AccessibilityView" Value="Raw"/>
             <Setter Property="IsTabStop" Value="False"/>
             <Setter Property="Width" Value="40"/>
-            <Setter Property="Height" Value="32"/>
```

Comment:
>Setter [](start = 13, length = 6)

Why is this getting deleted?

Reply:
This would set the height of the button to a fixed height making it look awkward I think. Since we scale now, setting height is not needed anymore.

Reply:
I think this works because it will size to the input TextBox now. Otherwise, I would normally expect to see a default height on an input control.